pkg/rbac/tokenReview_test.go

```
@@ -89,8 +89,26 @@ func Test_IsValidToken_expiredCache(t *testing.T) {
 
 }
+
+// Test_hashToken_keyIsHashed asserts that GetTokenReview stores entries under the
+// SHA-256 hash of the token, not the raw token string.
+func Test_hashToken_keyIsHashed(t *testing.T) {
+	mock_cache := newMockCache()
+	token := "super-secret-bearer-token"
+
+	// Trigger a TokenReview — the fake client returns an unauthenticated result,
+	// but a cache entry is still created.
+	mock_cache.GetTokenReview(context.TODO(), token)
```

golangci-lint (2.12.2)

[error] Error return value of `mock_cache.GetTokenReview` is not checked

(errcheck)